src/modules/statusUpdate.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
```

Comment:
- [ ] Kindly consider avoiding disabling linters because linter is a tool that helps developers write better code by checking for errors, potential bugs, and stylistic issues.

Reply:
I have defined the values in index.js where the file is imported